http/src/main/resources/reference.conf

```diff
@@ -1,14 +1,16 @@
+# NOTE: commented fields are overridden in firecloud-develop with templated values
+
+application {
+  applicationName = "leonardo"
+  #leoGoogleProject = "broad-dsde-dev"
+  leoServiceAccountJsonFile = "replace_me"
```

Comment:
should the ones with `replace_me` be commented out?

Reply:
They actually need to be there because they're referenced in other places like `${application.leoServiceAccountEmail}`… all references need to be resolvable before merging configs. I can add a comment about this.